src/htmlrenderer.cpp

```diff
@@ -91,11 +94,16 @@ void HtmlRenderer::render(std::istream& input,
 	const std::string& url)
 {
 	unsigned int image_count = 0;
+	unsigned int video_count = 0;
+	unsigned int audio_count = 0;
+	unsigned int source_count = 0;
```

Comment:
This isn't used. Should i remove it?

Reply:
Yeah, I guess you should. It's easy to see that \<video>/\<audio> lacks sources (by inspecting the raw RSS/Atom), and there's nothing we can do about those missing tags inside Newsboat; so let's get rid of useless code.